src/analyses/custom_bitvector_analysis.cpp

```diff
@@ -380,6 +380,18 @@ void custom_bitvector_domaint::transform(
             }
           }
         }
+        else if(identifier=="memcpy")
```

Comment:
Why is special-casing for memcpy appropriate, given that we ship a library implementing memcpy?